deltachat-jsonrpc/src/api/types/chat_list.rs

```diff
@@ -49,7 +49,6 @@ pub enum ChatListItemFetchResult {
         dm_chat_contact: Option<u32>,
         was_seen_recently: bool,
         last_message_type: Option<MessageViewtype>,
-        last_message_id: Option<u32>,
```

Comment:
I didn't get why it was added in https://github.com/deltachat/deltachat-core-rust/pull/4311 with the comment "addresses link2ext comment". Didn't find any related comment. @meganoahj, @Simon-Laux, why is it needed?

Reply:
to show icon for webxdc messages

Reply:
Ok, i put it back here, but now it's returned from `Chatlist::get_summary2()`

Reply:
I currently don't care about the internals, as long as the api for desktop stays the same. ;)